cleanupBuckets.js

```diff
@@ -0,0 +1,160 @@
+const http = require('http');
+const async = require('async');
+const AWS = require('aws-sdk');
+const { Logger } = require('werelogs');
+
+const log = new Logger('s3utils::emptyBucket');
+// configurable params
+const BUCKETS = process.argv[2] ? process.argv[2].split(',') : null;
+const ACCESS_KEY = process.env.ACCESS_KEY;
+const SECRET_KEY = process.env.SECRET_KEY;
+const ENDPOINT = process.env.endpoint;
+if (!BUCKETS || BUCKETS.length === 0) {
+    log.fatal('No buckets given as input! Please provide ' +
+        'a comma-separated list of buckets');
+    process.exit(1);
+}
+if (!ENDPOINT) {
+    log.fatal('ENDPOINT not defined!');
+    process.exit(1);
+}
+if (!ACCESS_KEY) {
+    log.fatal('ACCESS_KEY not defined');
+    process.exit(1);
+}
+if (!SECRET_KEY) {
+    log.fatal('SECRET_KEY not defined');
+    process.exit(1);
+}
+const LISTING_LIMIT = 300;
```

Comment:
why not just rely on the default 1000?

Reply:
Yeah it can be 1000, originally I put it lower because when there are a billion objects to delete it can overwhelm the cluster.